hack/sync_to_vm.sh

```diff
@@ -14,6 +14,12 @@ mkdir -p "./build/rootfs"
 
 podman run --rm --name $pkg -v $top_src_dir:/$pkg $pkg make install DESTDIR=/$pkg/build/rootfs
 
+# Remove console-login-helper-messages from VM if already exists
+ssh $ssh_opts -i $sshkey_path -p $ssh_port root@localhost <<EOF
+set -xeuo pipefail
+dnf remove -y console-login-helper-messages
+EOF
```

Comment:
Does this work:

```suggestion
ssh $ssh_opts -i $sshkey_path -p $ssh_port root@localhost \
    dnf remove -y console-login-helper-messages
```

If so it might look a little cleaner.